dicom_app/dicom_reader.py

```diff
@@ -0,0 +1,25 @@
+import pydicom
+from pydicom.data import get_testdata_files
+
+from PIL import Image
+from PyQt5.QtGui import QImage
+import matplotlib.pyplot as plt
+
+
+def convert_dicom_to_QImage(cv_img):
+    height, width = cv_img.shape
+    bytes_per_line = 3 * width
```

Comment:
Magic number, at least, there should be comment explaining why width is multiply by 3